libc/src/__support/fixed_point/fx_rep.h

```diff
@@ -45,6 +45,7 @@ template <> struct FXRep<short fract> {
   LIBC_INLINE_VAR static constexpr int FRACTION_LEN = SFRACT_FBIT;
   LIBC_INLINE_VAR static constexpr int TOTAL_LEN =
       SIGN_LEN + INTEGRAL_LEN + FRACTION_LEN;
+  LIBC_INLINE_VAR static constexpr int VALUE_LEN = INTEGRAL_LEN + FRACTION_LEN;
```

Comment:
nit: Perhaps to reduce copying `VALUE_LEN` N times in each specialization, you could have a free function

```
template <typename T>
constexpr int GetValueLen() { return T::INTEGRAL_LEN + T::FRACTIONAL_LEN; }
```

Reply:
Resolved by [1d0488a](https://github.com/llvm/llvm-project/pull/114318/commits/1d0488a6fa8735b1bb8dc08a561eaf40e64ce581)